lib/generators/templates/email/email_interceptor.rb

```diff
@@ -0,0 +1,11 @@
+class EmailInterceptor
+  include ActiveSupport::Configurable
+
+  config_accessor :interceptor_addresses, default: []
+
+  def self.delivering_email(message)
+    to = interceptor_addresses
```

Comment:
Need to investigate if I should send an email per address in this list.

Reply:
Using an array works: https://guides.rubyonrails.org/action_mailer_basics.html#sending-email-to-multiple-recipients